packages/graphql/src/services/questions.ts

```
@@ -695,6 +720,24 @@ export async function updateElementInstances(
           microLearningId: undefined,
+          groupActivityId: undefined,
+        },
+      ]
+    } else if (
+      instance.elementStack?.groupActivity?.status ===
+        DB.PublicationStatus.DRAFT ||
+      instance.elementStack?.groupActivity?.status ===
+        DB.PublicationStatus.SCHEDULED
+    ) {
+      return [
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)